src/containers/Client.tsx

```diff
@@ -55,75 +77,122 @@ export const Client = ({
   const api = client.useSelector((snapshot) => snapshot.connectivity.api);
   const jellyfishClient = client.useSelector((snapshot) => snapshot.connectivity.client);
   const { signalingHost, signalingPath, signalingProtocol } = useSettings();
-
+  const [activeOutgoingStreams, setActiveOutgoingStreams] = useState(new Map<string, MediaStream>());
   const [show, setShow] = useLocalStorageState(`show-json-${peerId}`);
-
-  const [trackId, setTrackId] = useState<string | null>(null);
+  const [expandedToken, setExpandedToken] = useState(false);
+  const [tracksId, setTracksId] = useState<(track | null)[]>([]);
   const [tokenInput, setTokenInput] = useState<string>("");
 
   const isThereAnyTrack =
     Object.values(fullState?.remote || {}).flatMap(({ tracks }) => Object.values(tracks)).length > 0;
-
   useLogging(jellyfishClient);
   useConnectionToasts(jellyfishClient);
+  const [maxBandwidth, setMaxBandwidth] = useState<string | null>(getStringValue("max-bandwidth"));
+  const [trackMetadata, setTrackMetadata] = useState<string | null>(getStringValue("track-metadata"));
+  const [attachMetadata, setAddMetadata] = useState(getBooleanValue("attach-metadata"));
+  const [simulcastTransfer, setSimulcastTransfer] = useState(getBooleanValue("simulcast"));
+  const [selectedDeviceId, setSelectedDeviceId] = useState<DeviceInfo | null>(
+    { id: getStringValue("selected-device-stream") || "", type: getStringValue("selected-device-type") || "" } || null,
+  );
+  const [activeStreams, setActiveStreams] = useState<DeviceIdToStream | null>(null);
+  const [currentEncodings, setCurrentEncodings] = useState(
+    (getArrayValue("current-encodings") as TrackEncoding[]) || ["h", "m", "l"],
+  );
+
+  const changeEncodingRecieved = (trackId: string, encoding: TrackEncoding) => {
+    if (!fullState) return;
+    {
+      api?.setTargetTrackEncoding(trackId, encoding);
+      console.log("changed encoding");
+    }
```

Comment:
You don't need these parenthesis:

```suggestion
      api?.setTargetTrackEncoding(trackId, encoding);
      console.log("changed encoding");
```